src/Examples/UnitModels/Reactors/plug_flow_reactor_testing.ipynb

```diff
@@ -0,0 +1,1147 @@
+    "\n",
+    "$ r = -kVC_{EO} $, $ k = k_0 e^{(-E_a/RT)}$, with the variables as follows:\n",
+    "\n",
+    "$r$ - reaction rate extent in moles of ethylene oxide consumed per second; note that the traditional reaction rate would be given by $rate = r/V$ in moles per $m^3$ per second  \n",
+    "$k$ - reaction rate constant per second  \n",
+    "$V$ - volume of PFR in $m^3$, note that this is *liquid volume* and not the *total volume* of the reactor itself  \n",
+    "$C_{EO}$ - bulk concentration of ethylene oxide in moles per $m^3$ (the limiting reagent, since we assume excess catalyst and water)  \n",
+    "$k_0$ - pre-exponential Arrhenius factor per second  \n",
+    "$E_a$ - reaction activation energy in kJ per mole of ethylene oxide consumed  \n",
+    "$R$ - gas constant in J/mol-K  \n",
+    "$T$ - reactor temperature in K\n",
+    "\n",
+    "These calculations are contained within the property, reaction and unit model packages, and do not need to be entered into the flowsheet. More information on property estimation may be found below:\n",
+    "\n",
+    "ParamEst parameter estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/workflow/data_rec_parmest.html?highlight=paramest  \n",
+    "HELMET thermodynamic estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/modeling_extensions/surrogate/helmet/index.html  \n",
+    "RIPE reaction estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/modeling_extensions/surrogate/ripe/index.html  \n",
+    "\n",
+    "Let us import the following modules from the same directory as this Jupyter notebook:\n",
+    "- egprod_ideal as thermo_props\n",
+    "- egprod_reaction as reaction_props"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import egprod_ideal as thermo_props\n",
+    "import egprod_reaction as reaction_props"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Constructing the Flowsheet\n",
+    "\n",
+    "We have now imported all the components, unit models, and property modules we need to construct a flowsheet. Let us create a ConcreteModel and add the flowsheet block. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m = ConcreteModel()\n",
+    "m.fs = FlowsheetBlock(default={\"dynamic\": False})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now need to add the property packages to the flowsheet. Unlike Module 1, where we only had a thermo property package, for this flowsheet we will also need to add a reaction property package. We will use the Generic Property and Generic Reaction Frameworks; more information may be found on these methods at https://idaes-pse.readthedocs.io/en/1.8.0/user_guide/components/property_package/index.html."
```

Comment:
Again, it would be nice to link to Module 1 here if possible.

Reply:
See earlier comment on embedding links in Markdown.